config/secrets.yml.example

```diff
@@ -147,7 +147,7 @@ test:
     cookie:
       name: ox
       options:
-        domain: ''
+        domain: :all
         secure: false
```

Comment:
I think you don't want the leading `:` on all.  Since this is a yaml file you can't use ruby symbols.  Maybe it's better to just let domain be unset here which would be `nil` when it's read in?

Reply:
Ah, I mindlessly copied the change I'd made in the ERB context for development, into the static test context. Will fix.

Reply:
ok changed it to make the domain attribute completely conditional - don't set it if it's not requested.